### PR TITLE
Matched a few functions, and updated some names / types of variables and other functions

### PR DESCRIPTION
**Pull request by Ryan-Myers**
Figured I should just make a PR for this before it gets lost. It's mostly centered around menu.c function matches.

### CODE REVIEW

File: src/menu.c [hunk 1]
```diff
@@ -1531,98 +1531,98 @@ void load_menu_text(s32 language) {
...
-        gGameSelectTextElemsWithAdv2[3].unk14_a.asciiText = gMenuText[142]; // ADVENTURE
-        gGameSelectTextElemsWithAdv2[5].unk14_a.asciiText = gMenuText[143]; // ADVENTURE TWO
-        gGameSelectTextElemsWithAdv2[7].unk14_a.asciiText = gMenuText[144]; // TRACKS
-        sBadControllerPakMenuText[0] = gMenuText[69];                       // BAD CONTROLLER PAK
-        sControllerPakFullMenuText[0] = gMenuText[68];                      // CONTROLLER PAK FULL
-        sNoControllerPakMenuText[0] = gMenuText[67];                        // NO CONTROLLER PAK
-        sCorruptDataMenuText[4] = 0;
-        sCorruptDataMenuText[5] = 0;
-        sCorruptDataMenuText[0] = gMenuText[184];                           // CORRUPT DATA.
+            gOptionMenuStrings[1] = gMenuText[ASSET_MENU_TEXT_SUBTITLESOFF];                         // SUBTITLES OFF
```
Comment (AntonioCastelli) [inline src/menu.c:1559]: Do you mind wrapping these comments in quotes (e.g. `// "SUBTITLES OFF"`) just so it's more obvious that it represents the string in `gMenuText` and not a reiteration of the `AssetMenuTextIdsEnum`?

File: src/game_text.c [hunk 2]
```diff
@@ -94,26 +94,26 @@ void func_800C2F1C(s32 arg0) {
...
-            if (D_8012A7A8 >= 0x101) {
-                D_8012A7A8 = 0x100;
+            sDialogueAlpha += arg0 * D_8012A7AA;
+            if (sDialogueAlpha > 256) {
+                sDialogueAlpha = 256;
             }
             D_8012A7AC -= arg0;
             if (D_8012A7AC <= 0) {
-                func_800C2D6C(D_8012A7B6);
+                func_800C2D6C();
```
[the remark this answers is not in the document]
Comment (Ryan-Myers) [reply src/game_text.c:111]: Yeah, to the best of my ability, that function doesn't appear to take any arguments, and it was just coincidence that D_8012A7B6 was in a0 at that time.

File: src/menu.c [hunk 3]
```diff
@@ -511,7 +510,8 @@ s16 D_800DFCAC[4] = {
     0x000B, 0x000C, 0x0002, 0xFFFF
 };
 
-MenuElement D_800DFCB4[7] = {
+//Should this only have 6? The final record seems to be unused and blank.
```
[the remark this answers is not in the document]
Comment (Ryan-Myers) [reply src/menu.c:513]: Good call. I've removed that comment.

File: src/menu.c [hunk 4]
```diff
@@ -1495,7 +1495,7 @@ const char D_800E8250[] = "loadFrontEndItem() - Item no %d out of range 0-%d\n";
 
 /*********************************/
 
-#ifdef NON_EQUIVALENT
+#if 0
```
[the remark this answers is not in the document]
Comment (Ryan-Myers) [reply src/menu.c:1498]: That was my bad. I was working on it, and forgot to revert this.

File: src/menu.c (hunk 3 above)
[the remark this answers is not in the document]
Comment (AntonioCastelli) [reply src/menu.c]: You can leave a comment staying it's to null-terminate the list if it's not obvious. Up to you.

**Review by AntonioCastelli — APPROVED** [review]

**Pull request merged**